SPECS/openssh/openssh.spec

```diff
@@ -143,6 +143,19 @@ sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin no/' %{buildroot}%{
 sed -i 's/#PrintMotd yes/PrintMotd no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#PermitUserEnvironment no/PermitUserEnvironment no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#ClientAliveInterval 0/ClientAliveInterval 120/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i 's/#ClientAliveCountMax 0/ClientAliveCountMax 0/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
```

Comment:
These probably should stay as `sed` commands we are pulling in the rpm macros into the paths and modifying the file after its placed into the install environment.

Reply:
I've reached out to Henry to see if there is an important reason we did it this way.

Reply:
Henry doesn't recall, probably just because we had some other stuff done with way.

My personal opinion is patch files are better, but I'll leave it to you.